### ISSUE TITLE
Visual Rework of VPP website

### ISSUE THREAD

**Issue by edwarnicke** [FDio/site#112]
@nealtex01 has kindly commissioned some mock ups of a visual rework of the FD.io website:

https://docs.google.com/presentation/d/1928l56YGFk9oBuWaAx0FRzZBUUnGnIJDOxb1qMzaj-c/edit#slide=id.g100f35f23ab_2_0

@otroan @sknat Thoughts?

**Comment by edwarnicke** [comment]
![image](https://user-images.githubusercontent.com/1455208/140949748-59459c9e-8d14-461c-bde7-794ab6cfc6af.png)

**Comment by sknat** [comment]
Definitely like the new look.
The part which imho is worth discussing is the header content :
Today we have (among which "Get Started" can - I think - be removed)
````
Use Cases
    Use Cases
    Solutions
    User Feedback
Get Started   --|
    Technology  |
    Installing  |= For this we could reply on the VPP sphinx
    Building  --|
    Videos    --> Move under "Use Cases" ?
Documentation
    VPP documentation
    CSIT documentation
    HICN documentation
    Other projects
The Latest
    Latest news
    Webinars
    Whitepapers
Get Involved
    Community
    Technical Steering Commitee (TSC)
    TSC Documentation
````

This maps to the proposed header :
`Use Cases -- Resource -- Docs -- News -- Community -- Charter -- About`
* `Resource` can be removed as `Get started` previously
* `Charter` : do we really need it here, can't it go under Community ?
* `About` : not sure what this would point to 

Finally about the links in the middle of the page, I'd say :
````
Learn about VPP  -> sphinx doc
See Use Cases    -> Use Cases > Use Cases
Get the code     -> VPP sphinx doc > download VPP page
View the docs    -> VPP sphinx doc
Join the Project -> Get involved > Community
````

**Comment by edwarnicke** [comment]
Maybe something more like this:

```
Use Cases
    Use Cases
    Solutions
    User Feedback
Get Started   --|
    Technology  |
    Installing  |= For this we could reply on the VPP sphinx
    Building  --|
    Videos    --> Move under "Use Cases" ?
Documentation
    VPP documentation
    CSIT documentation
    HICN documentation
    Other projects
The Latest -> Rename to News
    Latest news -> Rename to Press
    Webinars
    Whitepapers
    Blogs -> Optional new section when we get Blogs
Get Involved -> Rename to Community
    Community -> This becomes the landing page for Community (_index.md)
    Technical Steering Commitee (TSC)
    TSC Documentation
```

So at top level we get
``` Use Cases --- Docs --- News -- Community```

@sknat @nealtex01 Thoughts?

**Comment by sknat** [comment]
Looks good to me !

**Comment by nealtex01** [comment]
I'm totally ok with the above, but for sake of completeness, see the entirety of this deck.

By studying compatriot sites, and scoring what others did, I arrived at the longer nav:

https://docs.google.com/presentation/d/1OPqjiJYdtaEUjWqRYAk27ctbZg-OiZktSNF12hXq7J4/edit#slide=id.ge89333392d_0_40

I think the charts Ed sent earlier maybe didn't include all that background?

Anyway, we can clear it down at today fd.io noon call.

**Comment by edwarnicke** [comment]
@nealtex01 I agree, you did a lot of good background there... I sort of jumped to your conclusions on the visual stuff... @sknat the deck @nealtex01 linked to is well worth the read :)

**Comment by nealtex01** [comment]
@edwarnicke for even the Nav you laid out, where would you stay we are on the site copy/content front:

1. We have all the content we need on the existing site. It just needs to get moved around.
2. We have all the content we need on the existing site. But it needs to be 'rewritten' and moved around.
3. We are missing whole cloth in areas.

???

**Comment by sknat** [comment]
Definitely an interesting read to see the various design choices others have made ! 
- On the last question @nealtex01 I'd say (2), we have most of the material, but some might be worth moving around & rewriting a bit.
So I think we're mostly aligned. I'll see if I get cycles to throw at it this week to draft a first PR.

**Comment by sknat** [comment]
I guess we can now close this issue. Let's re-open a new one with the evolution from the current new design
